front/app/services/permissions/rules/routePermissions.ts

```diff
@@ -30,7 +30,7 @@ export const isModeratedProjectRoute = (
   item: IRouteItem,
   user: IUser | null
 ) => {
-  const idRegexp = /^\/admin\/projects\/([a-z0-9-]+)\//;
+  const idRegexp = /^\/admin\/projects\/([a-z0-9-]+)\/?/;
```

Comment:
Iva, some context: optional trailing slash. This caused the bug for project mods when they clicked the project Edit button in /admin/projects. Their URL didn't qualify because of this regex.